fastlane.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'credentials_manager', '>= 0.14.0', '< 1.0.0' # Password Manager
   spec.add_dependency 'spaceship', '>= 0.19.4', '< 1.0.0' # communication layer with Apple's web services
+  spec.add_dependency 'xcov', '>= 0.1', '< 1.0' # Needed to generate coverage reports
```

Comment:
Please don't introduce new dependencies, add `Actions.verify_gem!('xcov')` to your action implementation in the `run` method before requiring your gem